app/actions/local/channel.ts

```diff
@@ -193,7 +194,7 @@ export async function markChannelAsViewed(serverUrl: string, channelId: string,
     }
 }
 
-export async function markChannelAsUnread(serverUrl: string, channelId: string, messageCount: number, mentionsCount: number, lastViewed: number, prepareRecordsOnly = false) {
+export async function markChannelAsUnread(serverUrl: string, channelId: string, messageCount: number, mentionsCount: number, lastViewed: number, urgentMentionCount: number, prepareRecordsOnly = false) {
```

Comment:
the fact that `urgentMentionCount` is not optional, I'm thinking we should move it before `lastViewed` so that arguments are kinda "grouped" together.

another option would be something like

`export async function markChannelAsUnread(serverUrl: string, args: UnreadArgs, prepareRecordsOnly = false) {`

where you would have
```
type UnreadArgs = {
    channelId: string;
    lastViewed: number;
    messageCount: number;
    mentionsCount: number;
    urgentMentionCount: number;
}
```

what do you think ?

Reply:
Yup, we can do these changes, it improves readability, and is more maintainable long-term.